docs/platform/what-is-tizen/versions/tizen-8-0-m2.md

```diff
@@ -0,0 +1,134 @@
+# Tizen 8.0
+
+The second milestone (M2) release of Tizen 8.0 was announced in October 2023. Tizen is an open-source operating system (OS) maintained by Samsung, and it is being developed and optimized for smart devices since 2012.
```

Comment:
```suggestion
The second milestone (M2) release of Tizen 8.0 was announced in October 2023. Tizen is an open-source operating system (OS) maintained by Samsung, and it has been developed and optimized for smart devices since 2012.
```